### ISSUE TITLE
Loss spikes and per-class accuracy low on roads dataset

### ISSUE THREAD

**Issue by jakefleisig** [GeorgeSeif/Semantic-Segmentation-Suite#208]
Hi, I am having some issues doing road segmentation on a satellite image dataset.  I am training for 40 epochs on 1440 images with a batch size of 2, no data augmentation, the ResNet152 frontend and FC-DenseNet56 model. I have two classes, road and background, with roads labeled as black and background labeled as white.  I am using the RMS prop optimizer with a learning rate of 1e-4.  The version of TensorFlow I am using is 1.8.  

The problem I am having is such:
The loss does not decrease during training and the per-class validation accuracy for my roads seems very low (less than or equal to 1%).  I have tried lowering the learning rate (to 1e-5 and 1e-6) but when I do the loss creeps downward and then spikes back up.  When I increase the learning rate (to 1e-3 or 1e-2) the loss exponentially increases.   

What would you recommend I do?
